public/js/log.js

```diff
@@ -60,250 +59,295 @@ MAGI_log.resizes = {
 };
 MAGI_log.tooltips = [];
 
-$('#magi-loggingReadMore').click(function(e) {
-  e.preventDefault();
-  var isVisible = 'visible' == $('#magi-loggingReadMoreDiv').css('visibility'),
-      visibleState = isVisible ? 'hidden' : 'visible',
-      displayState = isVisible ? 'none' : 'block';
-
-  $('#magi-loggingReadMoreDiv').css('visibility', visibleState);
-  $('#magi-loggingReadMoreDiv').css('display', displayState);
-
+$().ready(function () {
+  $.get('/logEnabled', function(res, status) {
```

Comment:
Right. If the server doesn't respond (i.e., logging isn't enabled), or if
the user has disabled logging, the logging code wont' even initialize.

On Mon, May 11, 2015 at 9:44 PM Max Leiserson <notifications@github.com>
wrote:

> In public/js/log.js
> <https://github.com/raphael-group/magi/pull/4#discussion_r30098885>:
>
> > @@ -60,250 +59,295 @@ MAGI_log.resizes = {
> >  };
> >  MAGI_log.tooltips = [];
> >
> > -$('#magi-loggingReadMore').
>
> click(function(e) {
> > -  e.preventDefault();
> > -  var isVisible = 'visible' == $('#magi-loggingReadMoreDiv').css('visibility'),
> > -      visibleState = isVisible ? 'hidden' : 'visible',
> > -      displayState = isVisible ? 'none' : 'block';
> > -
> > -  $('#magi-loggingReadMoreDiv').css('visibility', visibleState);
> > -  $('#magi-loggingReadMoreDiv').css('display', displayState);
> > -
> > +$().ready(function () {
> > +  $.get('/logEnabled', function(res, status) {
>
> So you mean /logEnabled just returns false if the server is configured
> not to use logging?
>
> --
> Reply to this email directly or view it on GitHub
> <https://github.com/raphael-group/magi/pull/4/files#r30098885>.
>